src/index.tsx

```diff
@@ -141,16 +141,17 @@ export interface ReactTagsWrapperProps {
    * CSS class names for the component.
    */
   classNames?: {
-    root: string;
-    rootFocused: string;
-    selected: string;
-    selectedTag: string;
-    selectedTagName: string;
-    search: string;
-    searchInput: string;
-    suggestions: string;
-    suggestionActive: string;
-    suggestionDisabled: string;
+    tags: string,
+    tagInput: string,
+    tagInputField: string,
+    selected: string,
+    tag: string,
+    remove: string,
+    suggestions: string,
+    activeSuggestion: string,
+    editTagInput: string,
+    editTagInputField: string,
+    clearAll: string,
```

Comment:
Hey @chasemcdo thank you for the fix!
I was traveling so I couldn't check this one. I am back now.

The fix looks good, thanks for catching this! Even though this has been there for ages but none of them would have worked as these classes weren't used anywhere in the codebase

can you write a test for this? You can just verify if the class names are passed correctly to the dom element (taking snapshot)

Reply:
Added the test and also updated the README while I was at it, since we are running a yarn lockfile for dev, but the readme directed people to use npm.